hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFutureTest.java

```diff
@@ -129,22 +90,19 @@ public void isDone_whenObjectResponse() {
         DummyOperation op = new DummyOperation("foobar");
 
         InternalCompletableFuture future = operationService.invokeOnTarget(null, op, getAddress(local));
+
         assertTrue(future.isDone());
     }
 
-    // ========================= andThen ==================================
-
     // There is a bug: https://github.com/hazelcast/hazelcast/issues/5001
     @Test
     public void andThen_whenNullResponse_thenCallbackExecuted() throws ExecutionException, InterruptedException {
         DummyOperation op = new DummyOperation(null);
-
+        final ExecutionCallback callback = mock(ExecutionCallback.class);
```

Comment:
Unfortunately yes since it's used in the inner class. 
If we used some proxying mechanism that e.g. Awaitility provides we could remove most of the inner classes from our tests...